### DISCUSSION TITLE
Gitako vs OctoTree?

### DISCUSSION THREAD

**Discussion by YogiOnBioinformatics** [EnixCoda/Gitako#286] [Q&A]
Curious but what's the reason to use this tool over [OctoTree](https://chrome.google.com/webstore/detail/octotree-github-code-tree/bkhaagjahfmjljalopjnoealnfndnagc)?

Any info would be useful since I'm already an OctoTree user.

**Comment by iamqiz** [comment]
gitako is free and open source 


but octotree pro is not free
see https://www.octotree.io/pricing

![Snipaste_2023-04-13_13-11-48](https://user-images.githubusercontent.com/48077521/231659772-630e02bc-f5e3-4d07-9610-46037b258845.png)

**Reply by YogiOnBioinformatics** [reply]
Do you mind comparing it more so from a utility angle (i.e. the features)?

**Reply by iamqiz** [reply]
i used gitako just for less months, and I am octotree free user (I can not use octotree pro feature), so I can not compare them in detail

**Reply by YogiOnBioinformatics** [reply]
Fair enough. Thanks for letting me know.
